diff --git a/src/grapejuice_common/paths.py b/src/grapejuice_common/paths.py
index cd3870d..f697711 100644
--- a/src/grapejuice_common/paths.py
+++ b/src/grapejuice_common/paths.py
@@ -29,14 +29,6 @@ def local_share_locale() -> Path:
     return local_share() / "locale"
 
 
-def local_var() -> Path:
-    return dot_local() / "var"
-
-
-def local_log() -> Path:
-    return local_var() / "log"
-
-
 def xdg_config_home() -> Path:
     if "XDG_CONFIG_HOME" in os.environ:
         return _ensure_directory(Path(os.environ["XDG_CONFIG_HOME"]).resolve())
@@ -76,7 +68,7 @@ def application_manifest() -> Path:
 
 
 def logging_directory() -> Path:
-    return local_log() / "grapejuice"
+    return local_share_grapejuice() / "logs"
 
 
 def assets_directory() -> Path:
